_overviews/scala3-book/domain-modeling-tools.md

```diff
@@ -788,6 +882,83 @@ val d = IrishSetter("Big Red")   // "Big Red is a Dog"
 {% endtab %}
 {% endtabs %}
 
+### Overriding an implemented method
+
+A class can also override a method that is defined in a trait. Here is an example:
```

Comment:
This covers both `Overriding an implemented method` and `Mixing in multiple traits that have behaviors` parts from the Scala 2 version.